src/arrayMethodSort.js

```diff
@@ -4,8 +4,36 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
It's not recommended to modify the prototype of built-in JavaScript objects like Array. This can lead to unexpected behavior in other parts of the code that rely on the original behavior of these objects. Instead, consider creating a standalone function that implements the sorting functionality.

Reply:
it's change by npm lint not by me